### PR TITLE
chore(bug): #1205 fixed alignment issue on public project component

### LINKED ISSUE #1205: In public project card , there should be proper alignment of time
>  In public project card , there should be proper alignment of time . For e.g 2 days ago and 24 days ago are not aligned

From #1153

### PR DESCRIPTION
**Pull request by webkhushboo**
closes #1205
### Notes

- [ ] UI: Fixed alignment issue on public project component

### CODE REVIEW

File: web/src/app/shared/components/projects/projects.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.text-right{
+    text-align: right;
+    width: 100px;
+}
```
Comment (eddiejaoude) [inline web/src/app/shared/components/projects/projects.component.scss]: Missing blank file line ending

**Comment by eddiejaoude** [comment]
Can you add a screenshot please 📸

File: web/src/app/shared/components/projects/projects.component.scss (hunk 1 above)
Comment (webkhushboo) [reply web/src/app/shared/components/projects/projects.component.scss]: Thanks for pointing it out. Fixed it

**Comment by webkhushboo** [comment]
Public project page screenshot:
image

**Comment by eddiejaoude** [comment]
> `chore(bug): #1205 fixed alignment issue on public project component`

The commit message style should be `defect(projects): ...` just for future reference 🤓

**Comment by webkhushboo** [comment]
okay sure !! Will keep in mind from next time :)

**Comment by webkhushboo** [comment]
Private project page:
image

**Comment by eddiejaoude** [comment]
Looks good 👍

**Comment by webkhushboo** [comment]
Thanks

**Pull request merged**